RestaurantManagementSystem/RestaurantManagementSystem/src/UseCases/EmailConfirmation/EmailConfirmationHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using RestaurantManagementSystem.src.Core.Contracts;
+using RestaurantManagementSystem.src.Core.Entities;
+using RestaurantManagementSystem.src.Core.Exceptions;
+using RestaurantManagementSystem.src.Infrastructure.Services;
+
+namespace RestaurantManagementSystem.src.UseCases.Email_Confirmation
+{
+    public class EmailConfirmationHandler : IEmailConfirmationHandler
```

Comment:
Focus on the high level of this task, EmailConfirmation is not a use case it should be `VerifyRegistrationRequest`